libraries/AP_Parachute/AP_Parachute.cpp

```diff
@@ -98,7 +108,12 @@ void AP_Parachute::update()
     if (_enabled <= 0) {
         return;
     }
-
+    // check if the plane is sinking too fast and release parachute
+    if((_critical_sink > 0) && !_release_initiated && _is_flying) {
+        if(_sink_rate > _critical_sink) {
```

Comment:
why not combine these all into a single line?

Reply:
>     * copter should call set_sink_rate() so it works on copters (as its a common parameter)

I'm a bit confused where copter calcs its altitude variation to call set_sink_rate()

Reply:
sure!